include/flatbuffers/base.h

```diff
@@ -139,11 +139,21 @@
   #endif
 #endif // !defined(FLATBUFFERS_LITTLEENDIAN)
 
+// Current Version.
 #define FLATBUFFERS_VERSION_MAJOR 1
 #define FLATBUFFERS_VERSION_MINOR 11
-#define FLATBUFFERS_VERSION_REVISION 0
+#define FLATBUFFERS_VERSION_REVISION 1
+// Compatible Version.
+#define FLATBUFFERS_COMPATIBLE_MAJOR 1
```

Comment:
Not sure how I feel about introducing this. This means that if we do a release, we could decide that we don't bump `COMPATIBLE`, because we know that all changes are non breaking, even of the internal APIs. First of all I don't think I will be ever be able to make that claim, because knowing that you didn't break something in the last 100 commits (typical for a release) for me seems impossible. Second, even if I did know, I would only know this for one language, and this version is potentially used across languages, whose API evolution is independent. So I don't understand when we'd ever use this. Maybe better to keep one version for now?

Reply:
Agree.